packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -134,31 +140,42 @@ export const TestProviders = ({
     )
 
     return (
-        <IntlProvider locale={locale.id} defaultLocale={DEFAULT_LOCALE} messages={messages}>
-            <MultiSiteProvider site={site} locale={locale} buildUrl={buildUrl}>
-                <CommerceAPIProvider value={api}>
-                    <CategoriesProvider categories={initialCategories}>
-                        <CurrencyProvider currency={DEFAULT_CURRENCY}>
-                            <CustomerProvider value={{customer, setCustomer}}>
-                                <BasketProvider value={{basket, setBasket}}>
-                                    <CustomerProductListsProvider>
-                                        <Router>
-                                            <ChakraProvider theme={theme}>
-                                                <AddToCartModalContext.Provider
-                                                    value={addToCartModal}
-                                                >
-                                                    {children}
-                                                </AddToCartModalContext.Provider>
-                                            </ChakraProvider>
-                                        </Router>
-                                    </CustomerProductListsProvider>
-                                </BasketProvider>
-                            </CustomerProvider>
-                        </CurrencyProvider>
-                    </CategoriesProvider>
-                </CommerceAPIProvider>
-            </MultiSiteProvider>
-        </IntlProvider>
+        <ServerContext.Provider value={{}}>
+            <IntlProvider locale={locale.id} defaultLocale={DEFAULT_LOCALE} messages={messages}>
+                <CommerceApiProvider
+                    shortCode={commerceApiConfig.parameters.shortCode}
+                    clientId={commerceApiConfig.parameters.clientId}
+                    organizationId={commerceApiConfig.parameters.organizationId}
+                    siteId={site?.id}
+                    locale={locale.id}
+                    redirectURI={`${window.location.origin}/testcallback`}
+                >
+                    <MultiSiteProvider site={site} locale={locale} buildUrl={buildUrl}>
+                        <CommerceAPIProvider value={api}>
```

Comment:
Nit: Can we rename this one to <_CommerceAPIProvider> to make it more distinct from the new commerceApi Provider?